packages/sitecore-jss/src/dictionary-service.ts

```diff
@@ -4,7 +4,15 @@ import { fetchData } from './dataApi';
 import { DictionaryPhrases, DictionaryServiceData } from './dataModels';
 import { HttpJsonFetcher } from './httpClientInterface';
 
-export type DictionaryServiceConfig = {
+export interface DictionaryService {
```

Comment:
@ambrauer Let's imagine that tomorrow we will add GraphqlDictionaryService, will we place both implementations in same file, or will we split? For me sounds like better to split it, may be do: `src/dictionary/rest-dictionary-service.ts`, `src/graphql-dictionary-service.ts`, `src/shared.ts` - it will contain `DictionaryService` interface. And something similar for LayoutService implementation: `src/rest-layout-service.ts`, `src/graphql-layout-service.ts`

Reply:
@illiakovalenko I personally don't mind them all in a single file, but let's make the call when we get to implementing GraphQL implementations? From package / API standpoint, there will be no breaking change if we decide so split at that time.

Reply:
Agree, can discuss it later, not required for current implementation, it's my ideas